test/unit/bootstrap-specs.js

```diff
@@ -17,16 +17,15 @@ chai.use(chaiAsPromised);
 
 describe('AndroidBootstrap', function () {
   const systemPort = 4724;
-  let androidBootstrap = new AndroidBootstrap(systemPort),
-      adb = new ADB(),
-      uiAutomator = new UiAutomator(adb);
+  let adb = new ADB();
```

Comment:
Seems like it, yes.

On Mon, Jan 11, 2016 at 12:13 PM, Isaac A. Murchie <notifications@github.com
> wrote:

> In test/unit/bootstrap-specs.js
> <https://github.com/appium/appium-android-bootstrap/pull/13#discussion_r49372018>
> :
>
> > @@ -17,16 +17,15 @@ chai.use(chaiAsPromised);
> >
> >  describe('AndroidBootstrap', function () {
> >    const systemPort = 4724;
> > -  let androidBootstrap = new AndroidBootstrap(systemPort),
> > -      adb = new ADB(),
> > -      uiAutomator = new UiAutomator(adb);
> > +  let adb = new ADB();
>
> I don't think so. But it might not hurt. createAdb just sets the path
> right.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appium/appium-android-bootstrap/pull/13/files#r49372018>
> .
>